### PR TITLE
Optimize image element creation. Reduces creation cost by ~25-30%.

### PR DESCRIPTION
**Pull request by willeastcott**
Optimize image element creation. Reduces creation cost for image element components by about 25-30%.

An even better solution would be to share the pc.Mesh between all element components, but that's a lot more work.

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/framework/components/element/image-element.js [hunk 1]
```diff
@@ -519,36 +525,26 @@ Object.assign(pc, function () {
...
-                    it.element[pc.SEMANTIC_NORMAL].set(this._normals[i * 3 + 0], this._normals[i * 3 + 1], this._normals[i * 3 + 2]);
-                    it.element[pc.SEMANTIC_TEXCOORD0].set(this._uvs[i * 2 + 0], this._uvs[i * 2 + 1]);
-
-                    it.next();
-                }
-                it.end();
+                vb.unlock();
 
-                mesh.aabb.compute(this._positions);
+                mesh.aabb.setMinMax(pc.Vec3.ZERO, new pc.Vec3(w, h, 0));
```
Comment (daredevildave) [inline src/framework/components/element/image-element.js]: Is this right? Seems like you're BB will always have one corner at [0,0,0]?

Comment (willeastcott) [reply src/framework/components/element/image-element.js]: Well spotted. Fixed.

**Comment by willeastcott** [comment]
Hey @daredevildave - this test now fails:

https://github.com/playcanvas/engine/blob/master/tests/framework/components/element/test_imageelement.js#L425

This is because the ```_uvs``` property doesn't exist any more. Options:

1. I look at the data in the vertex buffer and compare (which is a bit involved.
2. I delete the test.

What would you like to see done here?

**Comment by daredevildave** [comment]
We probably don't need to ready the UVs? The test says it is checking if updateMesh is called, and it is testing that above. Either way, I don't really mind

**Comment by willeastcott** [comment]
OK, I've deleted that particular test.

**Pull request merged**